integration/spark-common/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessorStepOnSpark.scala

```diff
@@ -138,7 +138,7 @@ object DataLoadProcessorStepOnSpark {
 
       override def next(): CarbonRow = {
         val row =
-          new CarbonRow(SortStepRowUtil.convertRow(rows.next().getData, sortParameters))
+          new CarbonRow(sortStepRowUtil.convertRow(rows.next().getData))
```

Comment:
Maybe `SortStepRowUtil` is not a good name anymore, it is some kind of converter

Reply:
@jackylk yeah, actually in the further PRs (After #1606 ), I plan to rename it to `SortStepRowConverter`. This converter can also be used to directly write records to sort temp file.